metaflow/plugins/kfp/tests/flows/check_error_handling_flow.py

```diff
@@ -1,25 +0,0 @@
-from metaflow import Parameter, FlowSpec, step, Step
```

Comment:
why remove this test?

Reply:
This test somehow trigger an initialization of datastore that resulted in error. See last [failure with this test](https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-workflow/aip-metaflow-integration-testing/-/jobs/7767408#L217)
```
Data store error:
Datastore root not found. Specify with METAFLOW_DATASTORE_SYSROOT_S3 environment variable.
```

I do not want to set `METAFLOW_DATASTORE_SYSROOT_S3` in code to have anything zillow specific. I intend to merge this PR first and then seek to understand why datastore is needed in the first place (maybe because I used a parameter for run id), and how to workaround it.

Reply:
Fixed. A white space was missing in bash code.